modules/shortcodes/js/jetpack-email-subscribe.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ */
+/**
+
+/* global jQuery */
+/* jshint esversion: 5, es3:false */
+var JetpackEmailSubscribe = {
+	validateEmail: function( email ) {
+		var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+		return re.test( String( email ).toLowerCase() );
+	},
+	activate: function( blogId, domId, cssPrefix ) {
```

Comment:
If these parameters are editable by non-Editors/Admins, odd things can happen. I believe that was part of the point of https://github.com/Automattic/jetpack/commit/46b63a30bc852431eea1fc075a00a9e5365c9aca, but that commit seems to have been reverted.

Reply:
Amazing catch ! Thank you -  I rebased it badly.